core/new-gui/src/app/dashboard/admin/component/admin-user.component.html

```diff
@@ -0,0 +1,197 @@
+<div class="section-container subsection-grid-container">
```

Comment:
I see you put the new components and services under `dashboard/admin`. It is a bit weird to add a new `admin` directory here instead of putting the component and service directly in `dashboard/component` and `dashboard/service`, unless you plan to add more components in `admin` in the future?

Reply:
There will be more pages under the admin category in the future. Admin views for workflows, files, and projects are on the road map.